tests/lib/rules/semi-spacing.js

```diff
@@ -61,75 +61,91 @@ ruleTester.run("semi-spacing", rule, {
     invalid: [
         {
             code: "var a = 'b' ;",
-            errors: [ { message: "Unexpected whitespace before semicolon.", type: "VariableDeclaration", line: 1, column: 13 } ]
+            errors: [ { message: "Unexpected whitespace before semicolon.", type: "VariableDeclaration", line: 1, column: 13 } ],
+            output: "var a = 'b';"
         },
         {
             code: "var a = 'b',\nc = 'd' ;",
-            errors: [ { message: "Unexpected whitespace before semicolon.", type: "VariableDeclaration", line: 2, column: 9 } ]
+            errors: [ { message: "Unexpected whitespace before semicolon.", type: "VariableDeclaration", line: 2, column: 9 } ],
+            output: "var a = 'b',\nc = 'd';"
         },
         {
             code: "var a = function() {} ;",
-            errors: [ { message: "Unexpected whitespace before semicolon.", type: "VariableDeclaration", line: 1, column: 23 } ]
+            errors: [ { message: "Unexpected whitespace before semicolon.", type: "VariableDeclaration", line: 1, column: 23 } ],
+            output: "var a = function() {};"
         },
         {
             code: "var a = function() {\n} ;",
-            errors: [ { message: "Unexpected whitespace before semicolon.", type: "VariableDeclaration", line: 2, column: 3 } ]
+            errors: [ { message: "Unexpected whitespace before semicolon.", type: "VariableDeclaration", line: 2, column: 3 } ],
+            output: "var a = function() {\n};"
         },
         {
             code: "/^a$/.test('b') ;",
-            errors: [ { message: "Unexpected whitespace before semicolon.", type: "ExpressionStatement", line: 1, column: 17 } ]
+            errors: [ { message: "Unexpected whitespace before semicolon.", type: "ExpressionStatement", line: 1, column: 17 } ],
+            output: "/^a$/.test('b');"
```

Comment:
Please put output next to code so that its easy to see the difference.
Example: https://github.com/eslint/eslint/pull/3802/files